resources/views/user/overview.blade.php

```diff
@@ -5,22 +5,87 @@
         <article class="flex-center">
             <div class="flex-column left-side padding-inline-1rem padding-block">
                 <h3>Search user</h3>
-                <p class="light-text">Hier komen zoekvelden.</p>
+                <form action="{{ route('user.overview') }}" method="GET">
+
+                    <label for="username-filter">Username</label>
+                    <input id="username-filter"type="text" name="username">
+
+                    <label for="first-name-filter">First Name</label>
+                    <input id="first-name-filter"type="text" name="first_name">
+
+                    <label for="last-name-filter">Last Name</label>
+                    <input id="last-name-filter"type="text" name="last_name">
+
+                    <label for="role-filter">Role</label>
+                    <select id="role-filter" name="role">
+                        <option value="">Select All</option>
+                        @foreach ($roles as $option)
```

Comment:
kan je $option naar $role hernoemen. de naam beschrijft wat het is, niet waar het voor gebruikt wordt